pkg/services/store/entity_events.go

```diff
@@ -143,6 +111,18 @@ func (e *entityEventService) GetAllEventsAfter(ctx context.Context, id int64) ([
 	return evs, err
 }
 
+func (e *entityEventService) saveEvent(ctx context.Context, cmd SaveEventCmd) error {
```

Comment:
hmm.. this method is now used only for tests - should we just remove it and have a different setup in tests?

Reply:
Done in f8f6a24ddcce9e54979314b2bd2e22bb00d109d8